cookiecutter.json

```diff
@@ -7,9 +7,13 @@
   "package_name": "{{cookiecutter.project_slug.replace('-', '_')}}",
   "project_short_description": "A cookiecutter template with ARC recommendations.",
   "initialise_git_repository": true,
+  "deploy_docs_to_github_pages": false,
   "github_username": "{{cookiecutter.author_given_names.lower().replace(' ', '-')}}-{{cookiecutter.author_family_names.lower().replace(' ', '-')}}",
   "min_python_version": ["3.9", "3.10", "3.11", "3.12"],
   "max_python_version": ["3.12", "3.11", "3.10", "3.9"],
   "license": ["MIT", "BSD-3", "GPL-3.0"],
-  "funder": "JBFC: The Joe Bloggs Funding Council"
+  "funder": "JBFC: The Joe Bloggs Funding Council",
+  "__prompts__": {
```

Comment:
Oo what does this do? (I can guess)

Reply:
Yeah agreed. Was tempted to add for all the keys here, but thought it would be better to do that in a different PR. Would help with #305

Reply:
Relevant docs page (though as you say fairly self explanatory!) https://cookiecutter.readthedocs.io/en/stable/advanced/human_readable_prompts.html

Reply:
Could also be useful for #205